<?xml version="1.0" encoding="UTF-8" ?>
<launch>
  <arg name="port" default="/dev/ttyUSB0" />
  <arg name="veh" default=""/>
  <arg name="I_am_base_station" default="False" />
  <arg name="use_boat1" default="True" />
  <arg name="use_boat2" default="True" />
  <arg name="use_boat3" default="True" />
  <arg name="use_boat4" default="True" />
  <arg name="use_boat5" default="True" />
  <arg name="use_boat6" default="True" />
  <arg name="use_boat7" default="True" />
  



  <rosparam command="load" file="$(find xbee_communication)/config/hrvo_xbee.yaml" />

  <node name="xbee_server_laptop" pkg="xbee_communication" type="xbee_odom_master1.py" output="screen" required="true">
    <param name="port" value="$(arg port)" />
    <param name="I_am_base_station" value="$(arg I_am_base_station)" />
    <param name="use_boat1" value="$(arg use_boat1)" />
    <param name="use_boat2" value="$(arg use_boat2)" />
    <param name="use_boat3" value="$(arg use_boat3)" />
    <param name="use_boat4" value="$(arg use_boat4)" />
    <param name="use_boat5" value="$(arg use_boat5)" />
    <param name="use_boat6" value="$(arg use_boat6)" />
    <param name="use_boat7" value="$(arg use_boat7)" />


  </node>


</launch>
